src/test/java/edu/cornell/kfs/module/ar/document/validation/impl/CuTotalAmountBilledToDateExceedsTotalBudgetSuspensionCategoryTest.java

```diff
@@ -65,8 +56,7 @@ public void testSuspensionByBudgetAmountNullBudgetTotal() {
     }
     
     private void prepareContractsGrantsInvoiceDocument(KualiDecimal totalAmountBilledToDate, KualiDecimal budgetTotal) {
-        PowerMockito.suppress(PowerMockito.constructor(DocumentBase.class));
-        contractsGrantsInvoiceDocument = PowerMockito.spy(new ContractsGrantsInvoiceDocument());
+        contractsGrantsInvoiceDocument = Mockito.mock(ContractsGrantsInvoiceDocument.class);
```

Comment:
Can this be replaced with either the document itself or a spied version of the document?